Cognite.Simulator.Utils/SimulationRunnerBase.cs

```diff
@@ -276,6 +282,9 @@ public async Task Run(CancellationToken token)
                             startTime, 
                             e, 
                             token).ConfigureAwait(false);
+                        
+                        PublishSimulationRunStatus("", token);
```

Comment:
as a an alternative we could keep this status in memory and write it back on the heartbeat.
Pros:
1. less code to maintain (no need of UpsertItemInKVPSequence)
Cons:
1. not that granular, but we are only interested in the long-running calculations anyway

Wdyt?

Reply:
Yep, I thought about this as well. The problem I was facing was creating an in memory resource, that both the `ConnectorBase` and `SimulationRunnerBase` could access.

Reply:
@abdullah-cognite what's the problem exactly?